src/WikiService.Mappers/Db/Interfaces/IDbRubricMapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.WikiService.Models.Db;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests;
+
+namespace LT.DigitalOffice.WikiService.Mappers.Db.Interfaces
+{
+  [AutoInject]
+  public  interface IDbRubricMapper
```

Comment:
fix spases between "public" and  "interface"

Reply:
done